app/src/main/java/com/asfoundation/wallet/topup/TopUpSuccessFragment.kt

```diff
@@ -107,7 +96,7 @@ class TopUpSuccessFragment : DaggerFragment(), TopUpSuccessFragmentView {
   }
 
   override fun show() {
-    if (validBonus) {
+    if (bonus?.isNotBlank() == true) {
```

Comment:
This was set as nullable due to copy paste I think (when this was done before, since before this was actually nullable). Can you remove the “?”?  on this and the others?